app/controllers/hello_react_controller.rb

```diff
@@ -2,6 +2,13 @@ class HelloReactController < ApplicationController
   def contracts
     @contractor = Contractor.new("seeds/offers.json")
     @contractor.build_contracts
+    @user = {
+      name: "Landy Simpson",
+      course: "CSC104H1F",
+      filename: "#{Rails.root}/tmp/contracts/1342088432-CSC104H1F-contract.pdf",
+      email: "mingtsengeh@gmail.com"
+      }
+    test = CpMailer.contract_email(@user).deliver_now!
```

Comment:
[`deliver_now!`](http://api.rubyonrails.org/classes/ActionMailer/MessageDelivery.html#method-i-deliver_now-21) seems to be a pretty harsh method:
> Delivers an email without checking perform_deliveries and raise_delivery_errors, so use with caution.

we should probably use `deliver_now` instead.